src/extensions/default/HTMLCodeHints/HtmlAttributes.json

```diff
@@ -212,7 +212,8 @@
     "menu/type":          { "attribOption": ["context", "list", "toolbar"] },
     "ol/type":            { "attribOption": ["1", "a", "A", "i", "I"] },
     "script/type":        { "attribOption": ["text/javascript", "text/ecmascript", "text/jscript", "text/livescript", "text/tcl", "text/x-javascript", "text/x-ecmascript", 
-                                             "application/x-javascript", "application/x-ecmascript", "application/javascript", "application/ecmascript"] },
+                                             "application/x-javascript", "application/x-ecmascript", "application/javascript", "application/ecmascript",
```

Comment:
This one also has the somewhat arbitrary new lines.  But these were here before.  No need to fix.